build/common.go

```diff
@@ -0,0 +1,286 @@
+//+build mage
```

Comment:
I found out that Mage b0rks if you use this build tag in the imported package, even if it looks from the documentation as if it should be supported. I made a [PR](https://github.com/magefile/mage/pull/298) to rectify the issue, although it remains to see what they say.